src/item.cpp

```diff
@@ -1371,7 +1347,7 @@ std::string item::info( bool showtext, std::vector<iteminfo> &info ) const
             temp1 << _( "<good>preserves spoiling</good>, " );
         }
 
-        temp1 << string_format( _( "can store <info>%.2f liters</info>." ), c.contains / 4.0 );
+        temp1 << string_format( _( "can store <info>%.2f liters</info>." ), c.contains / 1000.0_ml );
```

Comment:
The same reason why it was `4.0` and not `4` in the old code: the result should be a `double`. `1000_ml` has `int` as value type and `1000.0_ml` has `double` as value type, thereby keeping the existing behavior.